### PR TITLE
Int 554 disable submit if cards not saved

### PR DESCRIPTION
**Pull request by saraOSI**
### Jira Story

- [Disable submit if cards not saved INT-554](https://osi-cwds.atlassian.net/browse/INT-554)

### Purpose
Disable submit if cards not saved

### CODE REVIEW

File: spec/features/screening/submit_screening_spec.rb [hunk 1]
```diff
@@ -22,6 +22,61 @@
...
+        within('.card', text: 'Narrative') { click_button 'Save' }
+        within('.card', text: 'Incident Information') { click_button 'Save' }
+        within('.card', text: 'Allegations') { click_button 'Save' }
+        within('.card', text: 'Worker Safety') { click_button 'Save' }
+        within('.card', text: 'Cross Report') { click_button 'Save' }
+        within('.card', text: 'Decision') { click_button 'Save' }
+        expect(page).to have_button('Submit', disabled: false)
+      end
+
+      scenario 'submit button is if all screening cards are saved, but person cards are not' do
```
Comment (VinhOSI) [inline spec/features/screening/submit_screening_spec.rb]: eh?

File: spec/features/screening/submit_screening_spec.rb [hunk 2]
```diff
@@ -22,6 +22,61 @@
...
+        end
+        within('.card', text: 'Narrative') { click_button 'Save' }
+        within('.card', text: 'Incident Information') { click_button 'Save' }
+        within('.card', text: 'Allegations') { click_button 'Save' }
+        within('.card', text: 'Worker Safety') { click_button 'Save' }
+        within('.card', text: 'Cross Report') { click_button 'Save' }
+        within('.card', text: 'Decision') { click_button 'Save' }
+        expect(page).to have_button('Submit', disabled: false)
+      end
+
```
Comment (tparkerosi) [inline spec/features/screening/submit_screening_spec.rb]: Missing 'disabled/enabled' in scenario name

File: spec/features/screening/submit_screening_spec.rb [hunk 3]
```diff
@@ -22,6 +22,61 @@
...
+        within('.card', text: 'Worker Safety') { click_button 'Save' }
+        within('.card', text: 'Cross Report') { click_button 'Save' }
+        within('.card', text: 'Decision') { click_button 'Save' }
+        expect(page).to have_button('Submit', disabled: true)
+        within('.card', text: "#{participant.first_name} #{participant.last_name}") do
+          click_button 'Save'
+        end
+        expect(page).to have_button('Submit', disabled: false)
+      end
+
```
Comment (tparkerosi) [inline spec/features/screening/submit_screening_spec.rb]: add submit button state to scenaro

File: spec/features/screening/submit_screening_spec.rb (hunk 1 above)
Comment (saraOSI) [reply spec/features/screening/submit_screening_spec.rb]: 👍

File: spec/features/screening/submit_screening_spec.rb (hunk 2 above)
Comment (saraOSI) [reply spec/features/screening/submit_screening_spec.rb]: :

File: spec/features/screening/submit_screening_spec.rb (hunk 3 above)
Comment (saraOSI) [reply spec/features/screening/submit_screening_spec.rb]: 👍

**Comment by VinhOSI** [comment]
lgtm

**Pull request merged**